src/app/pages/TopicPage/index.styles.jsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+
+const styles = {
+  main: ({ spacings, mq }) =>
+    css({
+      margin: `0 ${spacings.FULL}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        margin: `0 ${spacings.DOUBLE}rem`,
+      },
+    }),
+  inner: css({
+    maxWidth: '63rem',
+    margin: '0 auto',
+  }),
+  inline: ({ mq }) =>
+    css({
+      [mq.GROUP_4_MIN_WIDTH]: {
+        alignItems: 'center',
+        display: 'flex',
+      },
+    }),
+  title: ({ spacings, mq }) =>
+    css({
+      margin: `${spacings.TRIPLE}rem 0`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        margin: `${spacings.QUADRUPLE} 0`,
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        margin: '3rem 0',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        margin: '2.5rem 0 3rem 0',
```

Comment:
We generally try to avoid using spacings that aren't available in theme, the spacing should link back to what UX asked for when the component was built.

Are the constants here not available in `spacings`? https://github.com/bbc/simorgh/pull/10342/files#diff-92057634d5283330a6167da6a5a33b6e5d75b3fed7e09cc1de94f12e797702abL62

Reply:
The spacings should be the same. The aim here is to stop using the legacy psammead spacings. We could change this to e.g. `${spacings.TRIPLE x 2}` if we are only using the theme spacings moving forward. We could also extend `theme.spacings`.